packages/suite/src/components/suite/AppNavigation/index.tsx

```diff
@@ -199,117 +280,130 @@ const AppNavigation = ({ items, primaryContent }: Props) => {
     const itemsSecondaryWithExtra = itemsSecondary.filter(item => item.extra);
     const itemsSecondaryWithoutExtra = itemsSecondary.filter(item => !item.extra);
 
+    const sticky = isInView();
+    const selectedAccount = useSelector(state => state.wallet.selectedAccount.account);
+
     return (
-        <Wrapper ref={wrapper} settingsWrapper={routeName && routeName.startsWith('settings')}>
+        <Wrapper ref={wrapper} value={sticky} subRoute={routeName && isSubsection(routeName)}>
             {routeName && isSubsection(routeName) ? (
-                <Primary>
-                    <StyledBackLink onClick={() => goto('wallet-index', undefined, true)}>
-                        <StyledIcon icon="ARROW_LEFT" size={16} />
-                        <Translation id="TR_BACK" />
-                    </StyledBackLink>
-                </Primary>
+                <InnerWrap>
+                    <MenuHolder
+                        settingsWrapper={routeName && routeName.startsWith('settings')}
+                        maxWidth={maxWidth}
+                        border={routeName && isSubsection(routeName)}
```

Comment:
nit: this condition will always be true as it is wrapped in the very same condition on [L288](https://github.com/trezor/trezor-suite/pull/3958/files#diff-2ef59b26da87bd6c43a73664c07034672bc7aabce88a2bcf35b536a51b0bbce5R288)

Reply:
331a45e053a53457e46506ccff18f29dd5f5b5fc